spec/02-integration/03-db/13-cluster_status_spec.lua

```diff
@@ -71,5 +71,31 @@ for _, strategy in helpers.each_strategy() do
         assert.is_nil(err)
       end)
     end)
+
+    describe("cert_details", function()
+      it(":upsert()", function()
+        local p, err = db.clustering_data_planes:upsert({ id = "eb51145a-aaaa-bbbb-cccc-22087fb081db", },
+                                                 { config_hash = "a9a166c59873245db8f1a747ba9a80a7",
+                                                   hostname = "localhost",
+                                                   ip = "127.0.0.1",
+                                                   cert_details = {
+                                                    expiry_timestamp = 1897136778,
+                                                   }
+                                                 })
+
+        assert.is_truthy(p)
+        assert.is_nil(err)
+      end)
+
+      it(":update()", function()
+        -- this time update instead of insert
+        local p, err = db.clustering_data_planes:update({ id = "eb51145a-aaaa-bbbb-cccc-22087fb081db", },
+                                          { config_hash = "a9a166c59873245db8f1a747ba9a80a7",
+                                          cert_details = { expiry_timestamp = 1888983905, }
+                                          })
```

Comment:
Would you mind tidying up the formatting of this code before we merge?

Reply:
thanks for review, formatted